scr/hw5/Cat.java

```diff
@@ -0,0 +1,24 @@
+package hw5;
+
+class Cat extends Animal {
+    public static int count = 0;
+
+    public Cat(String N) {
+        super(N);
+        count++;
+    }
+
+    public void swim(int L) {
+        System.out.println("Cat " + name + " can`t swim ");
+
+    }
+
+    public void run(int L) {
+        if (L <= 200) {
+            System.out.print("Cat " + name + " can run ");
+            System.out.println(L + " m");
```

Comment:
you can use System.out.println("Cat " + name + " can run " + distance + "m"); instead of two lines of print